dbt_platform_helper/commands/codebase.py

```diff
@@ -62,9 +62,27 @@ def build(app, codebase, commit):
     help="The codebase name as specified in the platform-config.yml file. This can be run from any directory.",
     required=True,
 )
-@click.option("--commit", help="GitHub commit hash", required=True)
-def deploy(app, env, codebase, commit):
+@click.option("--ref", help="ECR image tag, commit hash, or branch name", required=False)
+@click.option(
+    "--commit",
+    help="(DEPRECATED) Use --ref instead to pass the ECR image tag, GitHub commit hash, or branch name.",
+    required=False,
+)
+def deploy(app, env, codebase, commit, ref):
```

Comment:
We're trying to use type hints more in the code. Can you add them here as well?
```suggestion
def deploy(app: str, env: str, codebase: str, commit: str, ref: str):
```